libsspl_sec/tests/acceptance/helpers/base64.c

```diff
@@ -25,7 +25,6 @@
 #include <math.h>
 #include <openssl/bio.h>
 #include <openssl/evp.h>
-BIO_METHOD* BIO_f_base64(void);
```

Comment:
so we dont need this explicit declaration now ?

Reply:
no